eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -507,6 +518,7 @@ if ($SubscriptionId -and ($originalSubscription -ne $SubscriptionId)) {
 
 try {
   DeleteOrUpdateResourceGroups
+  #DeleteSubscriptionDeployments
```

Comment:
I assume you wish to re-enable this before merging.

Reply:
Yep disabled for testing. We have hundreds of this right now so I'm manually reconciling since it takes a very long time with that quantity.